### ISSUE TITLE
Check on pyodbc==3.0.7 fails

### ISSUE THREAD

**Issue by sekrause** [sesh/piprot#46]
```
$ cat requirements.txt
pyodbc==3.0.7

$ piprot --verbatim requirements.txt
pyodbc==3.0.7 # Error checking latest version
```

However [pyodbc 3.0.10 exists on PyPi](https://pypi.python.org/pypi/pyodbc/3.0.10).

**Comment by kvdb** [comment]
I see the same behavior . Some packages are checked, some aren't:

    requests[security]==2.7.0 # Error checking latest version
    sh==1.11.0 # Error checking latest version
    trollius==2.0.0 # Error checking latest version

**Comment by sesh** [comment]
Hey there. Sorry about this one.

The `pyodbc` issue should be fixed in 0.9.5 (will release that today, just closing out a couple of things).

@kvdb looks like `sh` latest version is actually `1.11.0`, and `trollius` is just `2.0`
